goalie/mesh_seq.py

```diff
@@ -49,12 +49,13 @@ def __init__(self, time_partition, initial_meshes, **kwargs):
             take various types
         """
         self.time_partition = time_partition
-        self.fields = dict.fromkeys(time_partition.field_names)
-        self.field_types = dict(zip(self.fields, time_partition.field_types))
+        self.fields = {field.name: field for field in time_partition.fields}
+        self.field_data = dict.fromkeys(self.fields)
```

Comment:
So this initializes to a dict mapping field name to None? Could you add a comment - as not everyone knows what `fromkeys(some_dict)` does by head - and also explain what is meant to be stored as value?

Reply:
Indeed, I didn't know it either - the linter reported this. Sure, it probably does warrant a comment, thanks.

Reply:
Addressed in 80f9433.